photon-ml/src/main/scala/com/linkedin/photon/ml/util/DefaultIndexMap.scala

```diff
@@ -36,7 +41,9 @@ class DefaultIndexMap(@transient val featureNameToIdMap: Map[String, Int]) exten
     _idToNameMap.get(idx)
   }
 
-  override def getIndex(name: String): Int = featureNameToIdMap.getOrElse(name, IndexMap.NULL_KEY)
+  override def getIndex(name: String): Int = {
+    featureNameToIdMap.getOrElse(name, IndexMap.NULL_KEY)
+  }
```

Comment:
The previous way is correct according to the Scala style guide ([here specifically](http://docs.scala-lang.org/style/declarations.html#body)).